src/vw/build.sbt

```diff
@@ -0,0 +1,2 @@
+//> DependsOn: core
```

Comment:
I think after the refactor this file can be removed?

Reply:
done